.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: java
 jdk: oraclejdk8
 dist: trusty
-
-# Make the gradle wrapper executable
-before_install: chmod +x gradlew
+sudo: false
+script:
+  - bash gradlew check
```

Comment:
Why make the default `/.gradlew check` explicit? Also, what is 'bash' doing there?

Reply:
In case the file does not have execution permission